### ISSUE TITLE
Disable the user interface notification for bad server certification

### ISSUE THREAD

**Issue by tiffrobo** [zom/Zom-iOS#615]
![img_4969 2](https://user-images.githubusercontent.com/30634981/40325888-f3cad838-5d02-11e8-923c-8e285b6e5db4.JPG)

**Comment by n8fr8** [comment]
This is also happening to Rosa. Any progress?

**Comment by n8fr8** [comment]
@abeluck any thoughts on the server side of this possibly?

**Comment by N-Pex** [comment]
Yes, I am pretty sure I've fixed it (a new UUID was used every time, so instead of "reusing" the notification a new one was created). I just need to be sure that the fix is not affecting things to the other extreme, i.e. showing too few notifications because they are lumped together. Will try to do a new build soon, probably tomorrow.

**Comment by n8fr8** [comment]
Can you tell me more about when it is triggered? Is it generated by a connection to the push.zom.im server or the home.zom.im?

Let's talk on the scrum today.

**Comment by n8fr8** [comment]
(oops tomorrow!)

**Comment by N-Pex** [comment]
@n8fr8 I haven't actually tried to chase the cause, I've only fixed the problem (by faking a connection problem in the code, so that it tries to show a new notification when trying to reconnect). I've not seen this, but that's probably because I don't test on the home.zom.im server. I can give that a go tomorrow as well.

**Comment by tiffrobo** [comment]
Let's keep an eye out for this. @N-Pex Fixed the multiple push notifications and user should only see one, but we need to figure out if this is a false push notification and if so, triggered by what.

**Comment by n8fr8** [comment]
Can we just disable this notification? Too many false positives.

**Comment by N-Pex** [comment]
Please retest in build 134.

**Comment by tiffrobo** [comment]
So! Kaci was seeing this error, but I she has the app store version. I haven't seen it during testing build 134 or 135. 

Device: iPhone 6, iPhone 7
OS: 10.3.3, 11.2.2
App version: build 135
